includes/payment-methods/class-klarna-payment-method.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Class Affirm_Payment_Method
+ *
+ * @package WCPay\Payment_Methods
+ */
+
+namespace WCPay\Payment_Methods;
+
+use WC_Payments_Token_Service;
+use WC_Payments_Utils;
+use WCPay\MultiCurrency\MultiCurrency;
+
+/**
+ * Affirm Payment Method class extending UPE base class
+ */
+class Klarna_Payment_Method extends UPE_Payment_Method {
+
+	const PAYMENT_METHOD_STRIPE_ID = 'klarna';
+
+	/**
+	 * Constructor for link payment method
+	 *
+	 * @param WC_Payments_Token_Service $token_service Token class instance.
+	 */
+	public function __construct( $token_service ) {
+		parent::__construct( $token_service );
+		$this->stripe_id                    = self::PAYMENT_METHOD_STRIPE_ID;
+		$this->title                        = __( 'Klarna', 'woocommerce-payments' );
+		$this->is_reusable                  = false;
+		$this->icon_url                     = plugins_url( 'assets/images/payment-methods/klarna.svg', WCPAY_PLUGIN_FILE );
+		$this->currencies                   = [ 'USD', 'GBP', 'EUR' ];
+		$this->accept_only_domestic_payment = true;
+		$this->countries                    = [ 'US', 'GB', 'AT', 'DE', 'NL', 'BE', 'ES', 'IT' ];
```

Comment:
We miss ranges per currency here, here's [a sample](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/payment-methods/class-afterpay-payment-method.php#L33).

Reply:
There's nothing about ranges in [Stripe's docs for Klarna](https://stripe.com/docs/payments/klarna). 

There are ranges in [Stripe's docs for Afterpay](https://stripe.com/docs/payments/afterpay-clearpay#collection-schedule). 

So, I left ranges out purposely.

Reply:
5000 is the baseline, as we don't have tech. ability to fetch the one for the buyer we stick to the baseline.

Reply:
Thank you for pointing this out, implementing right on

Reply:
@kalessil Added in https://github.com/Automattic/woocommerce-payments/pull/7282/commits/1cec35ea2c13129320710ce3291a00071cf730df

Reply:
Fantastic, thank you @KarlisJ !